doc/rinterface.rst

```diff
@@ -348,6 +348,9 @@ can also be triggered indirectly, as some on the R functions will behave
 differently when run interactively compared to when run in the so-called
 *BATCH mode*.
 
+Interactive mode could be disabled by setting environment variable
+`R_DISABLE_INTERACTIVE_MODE` to `1`.
```

Comment:
The environment variable would be specific to `rpy2`. I'd prefer to have it prefixed with `RPY2_` rather than `R_`.